packages/components/src/card-old/README.md

```diff
@@ -0,0 +1,107 @@
+# Card
+
+Card provides a flexible and extensible content container.
+
+## Usage
+
+```jsx
+import { Card, CardBody } from '@wordpress/components';
+
+const Example = () => (
+	<Card>
+		<CardBody>...</CardBody>
+	</Card>
+);
+```
+
+## Props
```

Comment:
Should we list props that are "inherited" from other components?

For example:
- `<Card />` inherits from `SurfaceProps` — should we list any of the props from `<Surface />`?
- `<CardDivider />` is basically a re-export of `<Divider />`  — should we list any of the props from `<Divider />`?

Reply:
We should just link to the `surface` documentation I think rather than duplicating it (otherwise we'd have to know everywhere to update whenever we make a change to Surface).

Reply:
Good point. Done in https://github.com/WordPress/gutenberg/pull/32243/commits/9ba2635e1dc56e6a6e0aab3c9ec8e551e1be3f5d